### PR TITLE
[edje] Change EAPI symbols for EDJE_API

### PR DESCRIPTION
**Pull request by Coquinho**

### CODE REVIEW

File: src/lib/edje/edje_private.h [hunk 1]
```diff
@@ -74,17 +74,17 @@
 #ifdef _WIN32
 # ifdef EFL_BUILD
 #  ifdef DLL_EXPORT
-#   define EAPI __declspec(dllexport)
```
Comment (Coquinho) [inline src/lib/edje/edje_private.h:77]: This probably should be deleted to include `edje_api`

Comment (joaoantoniocardoso) [reply src/lib/edje/edje_private.h:77]: yeah it would make more sense

File: src/lib/edje/Edje_Edit.h [hunk 2]
```diff
@@ -14,17 +14,17 @@
 #ifdef _WIN32
 # ifdef EFL_BUILD
 #  ifdef DLL_EXPORT
-#   define EAPI __declspec(dllexport)
+#   define EDJE_API __declspec(dllexport)
```
Comment (Coquinho) [inline src/lib/edje/Edje_Edit.h:17]: woops, same here

**Comment by Coquinho** [comment]
@joaoantoniocardoso now it should better suit us

**Comment by Coquinho** [comment]
Hey @joaoantoniocardoso and @walac, there were changes after your approval, so please look again.

**Comment by Coquinho** [comment]
> Use `<edje_api.h>` for great good. :)

Done, @JPTIZ

**Comment by Coquinho** [comment]
@JPTIZ, It is still marked as if you requested changes, which blocks merge.

**Comment by JPTIZ** [comment]
> @JPTIZ, It is still marked as if you requested changes, which blocks merge.

Re-request my revision then :*

**Pull request merged**